.github/workflows/post-release.yml

```diff
@@ -0,0 +1,26 @@
+name: post-release
+on:
```

Comment:
Not sure you intended to include these .github files

Reply:
My bad have removed .github files ..